Scripts/Swift/iOSTemplateMaker/Sources/iOSTemplateMaker/Helpers/Terminal.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Echo.swift
+//
+//
+//  Created by MarkG on 9/7/24.
+//
+
+import Foundation
+import ANSITerminal
+
+enum WriteStyle {
+
+    case section
+    case success
+    case error
+    case warning
+}
+
+func write(_ text: String, style: WriteStyle) {
+    switch style {
+    case .section:
+        writeln("-------------------".green)
+        writeln(text.green)
+        writeln("-------------------".green)
+    case .success:
+        writeln(text.green)
+    case .error:
+        writeln(text.red)
+    case .warning:
+        writeln(text.yellow)
+    }
+}
+
+func writeAt(_ row: Int, _ col: Int, _ text: String) {
+    moveTo(row, col)
+    write(text)
+}
+
+func ask(
```

Comment:
Does this trim the user's input? We might add spaces at the leading/trailing without notice.

Reply:
Do you you mean here? 🙏

Reply:
This function takes the user's input, right? I'm thinking about trimming users' input when they enter the app name and bundle ID...

Reply:
We won't need to handle it here, the `onValidate` is used from outside to validate the input.
For example:
It will show an error if we input `com.jfc.bundleid `
```
bundleIdProduction = ask(
    "Which is the bundle ID for the production environment?",
    note: "Ex: com.example.project",
    onValidate: validatePackageName
)
```